src/SampleDataPage.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from "react";
+import { ExtensionDataService } from "VSS/SDK/Services/ExtensionData";
+
+interface ISampleDataPageState {
+    collection: string,
```

Comment:
Rename it to more specific, objective and KRs?

Reply:
The user inputs the collection name. So if you want to add a KR, you'd put KR (or whatever) in that field when inserting data.